pom.xml

```diff
@@ -1151,7 +1151,7 @@
         <carbon.analytics.common.version>5.2.1</carbon.analytics.common.version>
 
         <!-- APIM Component Version -->
-        <carbon.apimgt.version>6.5.34</carbon.apimgt.version>
+        <carbon.apimgt.version>6.5.56-SNAPSHOT</carbon.apimgt.version>
```

Comment:
I think we can't mention the snapshot version due to auto-release

Reply:
Fixed